### ISSUE TITLE
ui5-select: aria-required attribute is not allowed for role=button

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#2339]
**Describe the bug**
aria-required attribute is used for listbox element with role=button. The aria-required attribute is not allowed on this DIV.

**To reproduce**
1. Go to https://github.wdf.sap.corp/pages/xweb/goalmanagementx/app.html?mock#/goal-list
2. click 'Skip Tour' button
3. click more actions button on the top right of Goal Card
4. click 'Create Activity' button
5. Find the last ui5-select on the popup and you will see the issue

**Expected behavior**
Use valid markup and follow the rules of the ARIA specification. The WAI-ARIA aria-required property indicates that user input is required before submission.

**Screenshots**
![araia-required on role button](https://user-images.githubusercontent.com/13098801/95712104-bd91aa80-0c96-11eb-972b-ec517a7cd783.png)

**Context**
- UI5 Web Components version: 0.23.2
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Comment by dobrinyonkov** [comment]
Hello,

Both `aria-required `and `required` attributes are not allowed on elements with role="button".
A more appropriate role which supports aria-required attrribute should be set to `ui5-select` (ex. listbox).

For now this is planned to be done with https://github.com/SAP/ui5-webcomponents/issues/2485

Kind Regards,
Dobrin

**Comment by nnaydenow** [comment]
Hello @GongRichard,

The role of the .ui5-select-root is changed to combobox in order to other accessibility issues.

Regards,
Nayden

**Comment by GongRichard** [comment]
Hi @nnaydenow ,

This issue has been fixed, but involved a new issue. Required ARIA attribute is not present: aria-expanded.
![2021-03-01_17-38-28](https://user-images.githubusercontent.com/13098801/109479311-357d3780-7ab5-11eb-9e0a-39d64ba293e0.jpg)

Thanks,
Richard

**Comment by GongRichard** [comment]
Raise a new ticket to track the new issues
https://github.com/SAP/ui5-webcomponents/issues/2910
